<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWidgetQt</class>
 <widget class="QWidget" name="VideoWidgetQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>536</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>16</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QGraphicsView" name="view3"/>
   </item>
   <item row="1" column="0">
    <widget class="QGraphicsView" name="view2"/>
   </item>
   <item row="0" column="1">
    <widget class="QGraphicsView" name="view1"/>
   </item>
   <item row="0" column="0">
    <widget class="QGraphicsView" name="view0"/>
   </item>
  </layout>
  <action name="actionUse_Texture_Rectangle">
   <property name="text">
    <string>Use Texture Rectangle</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>displayTypeChanged(int)</slot>
  <slot>outputFormatChanged(int)</slot>
  <slot>textureRectangleChecked(int)</slot>
 </slots>
</ui>
